### PR TITLE
feat: delete a champion from user champion list + add/delete multiple champions at one time

### PR DESCRIPTION
**Pull request by dead-programmer**

### CODE REVIEW

**Comment by dead-programmer** [comment]
> They look good. Can you unit test them please?

No

File: run.py [hunk 1]
```diff
@@ -2,15 +2,24 @@
...
+
+    print(user_input_string)
+
+    for i in user_input_string:
+        if '-' in i:
+            team_comps.delete_champ(i)
+        else:
+            team_comps.add_champ(i)
+
     calculate.match(user_input, team_comps.tier_S)
```
Comment (blakepennington97) [inline run.py]: I feel like we should move this into another function, or at least rename this function. It does more than just get user input.

File: team_comps.py [hunk 2]
```diff
@@ -5,35 +5,47 @@
...
-    user_data.append(input)
-    print(user_data)
+def get_team_data():
+    pprint(team_data)
+    return team_data
+
+
+def add_champ(champ_name):
+    user_team.append(champ_name)
+    print(user_team)
```
Comment (blakepennington97) [inline team_comps.py]: Add error catching here that prevents the user from entering a blank string or a champion that doesn't exist in the data set. Ex: "", "dinx", etc

File: team_comps.py [hunk 3]
```diff
@@ -5,35 +5,47 @@
...
+
+
+def delete_champ(deletion_input):                            # -champion_name
+    champ_name_length = len(deletion_input) - 1              # length = len(champion_name)
+    user_team.remove(deletion_input[-champ_name_length:])    # remove("champion_name")
+    print(user_team)
+
```
Comment (blakepennington97) [inline team_comps.py]: Error catching as mentioned above, to handle champions that don't exist in the user_team list.

**Pull request merged**